framework/db/pgsql/JsonExpressionBuilder.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace yii\db\pgsql;
+
+use yii\db\ExpressionBuilderInterface;
+use yii\db\ExpressionBuilderTrait;
+use yii\db\ExpressionInterface;
+use yii\db\JsonExpression;
+use yii\db\Query;
+
+/**
+ * Class JsonExpressionBuilder builds [[JsonExpression]] for PostgreSQL DBMS.
```

Comment:
We'll be able to add another builder for MySQL, correct?

Reply:
Right